### PR TITLE
Add sol3 for project_euler problem_03

### PR DESCRIPTION
**Pull request by jwmneu**
Add sol3 for project_euler problem_03

My Firt PR~~

### CODE REVIEW

File: project_euler/problem_03/sol3.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+"""
+Problem:
+The prime factors of 13195 are 5,7,13 and 29. What is the largest prime factor
+of a given number N?
+
+e.g. for 10, largest prime factor = 5. For 17, largest prime factor = 17.
+"""
+
+
+def solution(n):
```
Comment (cclauss) [inline project_euler/problem_03/sol3.py]: please add type hints __def solution(n: int) -> int:__

File: project_euler/problem_03/sol3.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        ...
+    TypeError: Parameter n must be int or passive of cast to int.
+    """
+    try:
+        n = int(n)
+    except (TypeError, ValueError) as e:
+        raise TypeError("Parameter n must be int or passive of cast to int.")
+    if n <= 0:
+        raise ValueError("Parameter n must be greater or equal to one.")
+    prime = 1
```
Comment (cclauss) [inline project_euler/problem_03/sol3.py]: Prime seems to be an unused variable.

Comment (jwmneu) [reply project_euler/problem_03/sol3.py]: you are right, thank you!

**Comment by cclauss** [comment]
Thanks!

**Pull request merged**